internal/connector/test/integration/features/connector-multitenancy-api.feature

```diff
@@ -357,10 +357,12 @@ Feature: connector namespaces API
     When I DELETE path "/v1/kafka_connector_clusters/${connector_cluster_id}"
     Then the response code should be 204
     And the response should match ""
+    # wait for cluster reconcile to delete namespaces
+    And I sleep for 5 seconds
```

Comment:
Not to be addressed in this PR but wonder if instead of having this artificial sleep time, we can have an await alike functionalist, i.e: 

```
Background:
    Given I wait at most 5 seconds
    Given I poll delay is 250 ms

Scenario:
    ...
    And Wait until selection ".items[] | lenght"  for GET path "/v1/kafka_connector_clusters" matches "0"
```

This would allow to be a little bit more resilient about timing issues and eventually faster

Reply:
Replaced sleep with a "wait up to" step to check the status of the namespace.